configapp/configapp.py

```diff
@@ -109,6 +155,12 @@ def data_image(filename):
     return send_from_directory(directory, filename)
 
 
+@app.route('/image/captured/<path:img>')
+def captured_image(img):
+    img = img.split('/')
+    return send_from_directory(os.path.join(img_predict_dir, img[0]), img[1])
```

Comment:
This url is only called from ajax get request where I need to specify subdirectory and name of the image. Do you have any idea how to improve this?